packages/payment-detection/src/thegraph/superfluid.ts

```diff
@@ -0,0 +1,30 @@
+import { GraphQLClient } from 'graphql-request';
```

Comment:
could you check @yomarion's approach about multiple clients in his NEAR PR?

Reply:
We discuss it when I've created this file. I've felt it was less confusing for me this way but I will do as done in NEAR PR.